src/chat/channel_view.py

```diff
@@ -190,7 +190,7 @@ def build(cls, theme, player_colors, **kwargs):
 
     def _load_template(self):
         self._env = jinja2.Environment()
-        template_str = self._theme.readfile("chat/channel.css")
+        template_str = self._theme.readfile("chat/channel.jinja")
```

Comment:
Does it come with the file rename? Either way I think we should keep the .css suffix (or maybe add a double css.jinja one?)

Reply:
you said the css is unfortunate and that we should change that Q.Q and yes the file is renamed too

Reply:
I mean, .css is unfortunate for the tooltip, since it's not a css file in any way. The channel file is actually a template of a css file.

Reply:
ah, alright, so qthtml makes sense, i thought itd be a good way to have all jinja templates share one naming, but that makes sense too